### DISCUSSION TITLE
WebAssembly support

### DISCUSSION THREAD

**Discussion by niqo01** [aws/aws-sdk-kotlin#1222] [Ideas]
I was wondering if supporting WebAssembly as a Kotlin multiplatform target is something that is considered?

**Comment by aajtodd** [comment]
Indeed it's being [considered](https://github.com/awslabs/aws-sdk-kotlin/issues/1043) but no firm plans or commitment at the moment. Feel free to upvote the linked issue and expand on your use case.
